dev/com.ibm.websphere.appserver.features/visibility/public/jaxrs-3.0/io.openliberty.jaxrs-3.0.feature

```diff
@@ -0,0 +1,17 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jaxrs-3.0
+visibility=public
+singleton=true
+IBM-API-Package: com.ibm.websphere.jaxrs.server; type="ibm-api", \
```

Comment:
Perhaps I'm jumping ahead a bit but doesn't this package come from `com.ibm.ws.jaxrs.2.0.server` and that will need to be transformed?